pyrsistent/_pclass.py

```diff
@@ -139,6 +139,10 @@ def transform(self, *transformations):
         return transform(self, transformations)
 
     def __eq__(self, other):
+        if self is other:
+            return True
+        if hash(self) != hash(other):
+            return False
```

Comment:
There doesn't seem to be as much coverage of the following code now. See attached screenshots:
![screenshot from 2016-03-28 15-08-13](https://cloud.githubusercontent.com/assets/978965/14091761/0e9e14f2-f4f7-11e5-9d9a-60ed54bb6073.png)
![screenshot from 2016-03-28 15-08-02](https://cloud.githubusercontent.com/assets/978965/14091764/10df73c8-f4f7-11e5-8064-2026f3f794bf.png)